src/uct/ib/mlx5/ib_mlx5.inl

```diff
@@ -24,13 +31,11 @@ uct_ib_mlx5_poll_cq(uct_ib_iface_t *iface, uct_ib_mlx5_cq_t *cq)
     cqe    = uct_ib_mlx5_get_cqe(cq, index);
     op_own = cqe->op_own;
 
-    if (ucs_unlikely((op_own & MLX5_CQE_OWNER_MASK) == !(index & cq->cq_length))) {
+    if (ucs_unlikely(uct_ib_mlx5_no_cqe(op_own, index, cq->cq_length))) {
```

Comment:
this adds extra branch, maybe initialize the CQEs to op_own=0x1 instead of 0xF?

Reply:
you mean to `0`?